scripts/perf/run_centaur_blocking/create_runner_instance.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -euo pipefail
+
+source scripts/perf/helper.inc.sh
+
+# Prepare the startup script to run on new instance
+mkdir -p mnt
+cp scripts/perf/run_centaur_blocking/test_runner_script.sh mnt/
+
+
+read_service_account_from_vault
+
+metadata=(
+  "BUILD_NUMBER=${BUILD_NUMBER}"
+  "BUILD_TAG=${BUILD_TAG}"
+  "CLEAN_UP=true"
+  "CROMWELL_PROJECT=${CROMWELL_PERF_PROJECT}"
+  "CROMWELL_BUCKET=${GCS_BUCKET}"
+  "CROMWELL_PERF_SCRIPTS_BRANCH=${CROMWELL_BRANCH}"
+  "CROMWELL_UNDER_TEST=${CROMWELL_UNDER_TEST}"
+  "GCS_REPORT_PATH=${GCS_REPORT_PATH}"
+  "GCS_REPORT_BUCKET=${GCS_REPORT_BUCKET}"
+  "TEST_CASE_DIRECTORY=${TEST_CASE_DIRECTORY}"
+)
+
+# NB: The 'cromwell-perf' tag is required to allow us through cromwell's firewall
```

Comment:
Cromwell has a firewall?

Reply:
Yes but this comment is wrong - the tag actually makes sure Cromwell *gets given* the right set of firewall rules rather than gets allowed through someone else's